packages/ui/src/lib/appShell/AppShell.svelte

```diff
@@ -13,14 +20,44 @@
 	import { classNames } from '../utils/classNames';
 	import { getSetting } from './utils/getSettingByScreenWidth';
 
+	/**
+	 * Object expressing the sidebar position (`top`, `bottom`, `left`, `right`) at different screen sizes.
+	 * The keys are size breakpoints: for very small browser windows, the `initial` position will be used; for larger windows, the position corresponding to the largest breakpoint that is smaller than the window will be user.
+	 * Breakpoints should be ordered in ascending order of size.
+	 */
 	export let sidebarPlacement: SidebarPlacement = { initial: 'bottom', md: 'right' };
+
+	/**
+	 * If the sidebar is `alwaysOpen`, the sidebar will always be open and the close control will not be displayed.
+	 * As for the position, this can be responsive to the browser window size.
+	 * `sidebarAlwaysOpen` is an object where the keys are size breakpoints (or `initial`, corresponding to 0 pixels) and the values are Booleans indicating whether the sidebar is always open at that size.
+	 */
 	export let sidebarAlwaysOpen: { [key: string]: AlwaysOpenType } | undefined = undefined;
+
+	/**
+	 * If `true`, then opening the sidebar will push other content to the side (or up/down, depending on the sidebar position), rather than th sidebar being superimposed on the content.
```

Comment:
```suggestion
	 * If `true`, then opening the sidebar will push other content to the side (or up/down, depending on the sidebar position), rather than the sidebar being superimposed on the content.
```